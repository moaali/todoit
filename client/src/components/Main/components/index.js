export { default as Todos } from './Todos';
export { default as Header } from './Header';
export { default as LoadMoreButton } from './LoadMoreButton';
