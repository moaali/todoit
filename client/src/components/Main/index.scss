.main {
  // Box Model
  max-width: 1600px;
  margin: 0 auto;
  padding: 75px 15px 15px 15px;

  // Ease
  transition: all 180ms ease-in-out;

  @media (min-width: 576px) {
    // Box Model
    padding: 140px 30px 30px 30px;
  }

  @media (min-width: 992px) {
    // Box Model
    padding: 140px 60px 60px 340px;

    .collapsed & {
      // Box Model
      padding: 140px 60px 60px 60px;
    }
  }
}

.todos {
  // Box Model
  display: grid;
  grid-template-columns: 1fr;
  grid-gap: 15px;

  @media (min-width: 768px) {
    // Box Model
    grid-template-columns: 1fr 1fr;
  }

  @media (min-width: 992px) {
    // Box Model
    grid-template-columns: 1fr;
  }

  @media (min-width: 1200px) {
    // Box Model
    grid-template-columns: 1fr 1fr;
  }

  @media (min-width: 1600px) {
    // Box Model
    grid-template-columns: 1fr 1fr 1fr;
  }
}
