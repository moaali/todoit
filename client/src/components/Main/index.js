import React, { PureComponent } from 'react'

import { Todos, Header, LoadMoreButton } from './components'
import './index.scss'

class Main extends PureComponent {
  render() {
    return (
      <div id='main' className='main'>
        <Header />
        <Todos />
        <LoadMoreButton />
      </div>
    )
  }
}

export default Main;
