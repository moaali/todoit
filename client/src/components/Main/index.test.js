import React from 'react';

import Main from './';

describe('Main', () => {
  it('should match snapshot', () => {
    const mainElement = shallow(<Main />);
    expect(toJson(mainElement)).toMatchSnapshot();
  })
})
