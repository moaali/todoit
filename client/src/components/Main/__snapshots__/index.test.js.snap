// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should match snapshot 1`] = `
<div
  className="main"
  id="main"
>
  <Header />
  <inject-Todos-with-store />
  <inject-LoadMoreButton-with-store />
</div>
`;
